app/views/grant_submissions/submissions/index.xlsx.axlsx

```diff
@@ -0,0 +1,73 @@
+
+base_headers_row_2 = ["Applicant",
+                "Email",
+                "Submission Title",
+                "State",
+                "Last Updated",
+                "Completed Reviews",
+               ]
+
+question_headers_row_2  = []
+
+@questions.each do |question|
+  header_text = question.text
+  header_text = header_text + ' (Required)' if question.is_mandatory
+  question_headers_row_2 << header_text
+end
+
+headers_row_2 = (base_headers_row_2 + question_headers_row_2)
+
+wb = xlsx_package.workbook
+
+sheet_name = prepare_excel_worksheet_name(sheet_name: @grant.name)
+
+header_style = wb.styles.add_style({b: true})
+link_style   = wb.styles.add_style({fg_color: '0000FF', u: true})
+
+wb.add_worksheet(name: sheet_name) do |sheet|
+  sheet.add_row headers_row_2, :style => header_style
+
+  response_column_widths = []
+
+  @submissions.each_with_index do |submission, row_index|
+    applicant = submission.applicant
+    responses = submission.responses
+    response_columns = []
+
+    review_data = review_data(submission)
+    reviews_completed = "#{review_data.completed_review_count} / #{review_data.assigned_review_count}"
+
+    file_upload_links = {}
+    question_index = 0
+    @questions.map do |question|
+      file_upload_info = []
+      question_index += 1
+
+      response = responses.to_a.detect { |response| response.grant_submission_question_id == question.id }
+
+      if question.response_type == 'file_upload'
+        file_upload_links[question_index] = rails_blob_url(response.document, disposition: 'attachment')
+        response_columns << response.response_value.to_s
+      else
+        response_columns << response.response_value
+      end
+      response_column_widths << 45
```

Comment:
In the reviews export, this was meant to shorten/widen the width of the spreadsheet to account for comments. Since all widths are going to be the same, it may not be worth keeping this array

Reply:
This gets appended on the end of the the other widths, which are variable. Without this The columns can get very long depending on the question or answers.